### PR TITLE
 Create integration test suite and migrate appropriate tests from unit test suite #595

### PR DESCRIPTION
**Pull request by erangeles**

### CODE REVIEW

**Comment by notmessenger** [comment]
* The Unit tests that were moved to Integration tests should be deleted from `tests/unit/components/sl-panel-test.js`
* Unit tests should be added to assert default values of `heading` and `loading`

File: tests/integration/components/sl-panel-test.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        'Default rendered component has class "panel-default"'
+    );
+
+    assert.ok(
+        this.$( '>:first-child' ).hasClass( 'sl-panel' ),
+        'Default rendered component has class "sl-panel"'
+    );
+});
+
+test( 'Valid heading value renders panel-heading', function( assert ) {
```
Comment (notmessenger) [inline tests/integration/components/sl-panel-test.js]: Assert text of rendered heading versus value passed to it.

**Comment by notmessenger** [comment]
Integration test should test that content can be yielded

**Comment by notmessenger** [comment]
Integration test should assert existence of classes/structure of:

* panel-body
* sl-maskable-content
* sl-mask

**Comment by erangeles** [comment]
@notmessenger this is really to be reviewed. all comments addressed.

**Pull request merged**